support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -223,14 +218,15 @@ function getClickHandler(props: PropTypes, isCustomPrb: boolean) {
 			props.useLocalCurrency,
 		);
 
-		if (
-			isCustomPrb &&
-			isValid &&
-			props.paymentRequestObject.status === 'AVAILABLE'
-		) {
-			props.paymentRequestObject.paymentRequest.show();
-		} else if (!isCustomPrb && !isValid) {
+		if (!isValid) {
 			event.preventDefault();
+			return;
+		}
+
```

Comment:
This logic is a little confusing because of our branching based on if it's a custom PRB or not. The previous `if/else` block was doing:

```ts
if (isValidCustomPaymentRequest) {
   /* ... */
} else if (isInvalidNonCustomPaymentRequest) {
   /* ... */
}
```

To simplify that a little we now just have a single `isValid` check which returns early if it's not valid. Technically we only need to call `event.preventDefault` when it's not a custom PRB, but it's fine to call for a custom PRB too. 

I've then moved the call to `updateTotal` after this, so that we only every call it when the amount is valid.

Finally we then have the logic for showing the payment request interface for a custom PRB.